inz-demo/id-card/PrivateKeys/1.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<IssuerPrivateKey xmlns="http://www.zurich.ibm.com/security/idemix">
+   <Counter>1</Counter>
+   <ExpiryDate>1649822770</ExpiryDate>
+   <Elements>
+      <p>163497273463210981162818887977690910401499777543170159265182643984040020466102048155986753025671994622629600934309072864921620563728016380163041920994176418210927168889702839270144768978540763705960502267340599453368221866788560942469329738983109939980006333546361466648013007585470553360039793939137513484427</p>
+      <q>129658288931865398588910703835991326273460163074553932290216841515785318957651609837069596727190587290944828519275040552400305637025461040868147459146909164110362422872352329480880198000953895998650161231230159256811859448750015227697003751305261404749874988766937440549169820717155705235040500768965988681527</q>
+      <pPrime>81748636731605490581409443988845455200749888771585079632591321992020010233051024077993376512835997311314800467154536432460810281864008190081520960497088209105463584444851419635072384489270381852980251133670299726684110933394280471234664869491554969990003166773180733324006503792735276680019896969568756742213</pPrime>
+      <qPrime>64829144465932699294455351917995663136730081537276966145108420757892659478825804918534798363595293645472414259637520276200152818512730520434073729573454582055181211436176164740440099000476947999325080615615079628405929724375007613848501875652630702374937494383468720274584910358577852617520250384482994340763</qPrime>
+   </Elements>
+   <ECDSA>MHcCAQEEIGUryEnBJ6M6LsmkaQTTMb3NL+R7ZbHs/T5IjZP0rK3poAoGCCqGSM49AwEHoUQDQgAEX4L8FqPJCo6QUVXdUEcnnfKz0mU6onRJzfeYokqkNhr4cwtfhVzr0fvi1OWeT5lS8R73Fk5EVHNxapWMbjytKQ==</ECDSA>
+</IssuerPrivateKey>
```

Comment:
Was this a keypair generated by `irma issuer revocation keypair` command?

Reply:
No, by `irma issuer keygen`. In the latest version of irma server all the generated keys are revocation-enabled by default.

Reply:
Nice. So this falls into the new generated keypair settings.